content/chainguard/chainguard-images/how-to-use/use-with-openshift.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "How to Use Chainguard Containers with OpenShift"
+linktitle: "Using With OpenShift"
+type: "article"
+description: "Special considerations for using Chainguard Containers with OpenShift"
+date: 2025-06-17T08:49:31+00:00
+lastmod: 2025-06-17T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "OpenShift", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 008
+toc: true
+---
+
+In this guide, you'll find general instructions for how to get started using Chainguard Containers on Red Hat OpenShift Container Platform.
+
+[Red Hat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) is an application platform that orchestrates and manages your systems and resources. While it is based on open source software like Kubernetes, OpenShift includes a suite of applications with additional functionality that are configured to work together.
+
+Adding Chainguard Containers to your OpenShift deployment saves you the effort of CVE remediation and speeds up your security and compliance efforts.
```

Comment:
Should this be something like "migrating your containers to Chainguard Containers"? Or maybe a rephrase of some kind. Otherwise looks great (not a blocking comment)

Reply:
This is adding in the sense of there may only be the OpenShift deployment with no payload. It could also be in the sense of an addition to an existing deployment of multiple other containers. Another possibility is the migration to our containers from existing containers. So, for flexibility, I liked "adding". Let's see how it goes and we can change it later if needed.